src/Esfa.Vacancy.Api.Types/CreateApprenticeship/WageUnit.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace Esfa.Vacancy.Api.Types.CreateApprenticeship
+{
+    public enum WageUnit
+    {
+        NotApplicable = 1,
```

Comment:
What happened to 0 Unspecified???

Reply:
its possible to submit with WageUnit of 0 but it will fail validation which is by design.